src/ProgressOnderwijsUtils/ReferenceEqualityComparer.cs

```diff
@@ -14,9 +14,9 @@ public bool Equals(T one, T other)
             => ReferenceEquals(one, other);
 
         public int GetHashCode(T obj)
-            => RuntimeHelpers.GetHashCode(obj);
+            => RuntimeHelpers.GetHashCode(obj!);

```

Comment:
evt. comment toevoegen dat dit een leugen is als workaround voor lib mislabeling.

Als dat vaker voorkomt willen we denk ik een shorthand ervoor, want je wilt niet dat de lezer *echt* denkt dat het obj niet nullable is en vervolgens verkeerde aannames maakt.